src/main/kotlin/com/emberjs/hbs/HbsCompletionContributor.kt

```diff
@@ -15,13 +15,13 @@ class HbsCompletionContributor : CompletionContributor() {
     init {
         extend(CompletionType.BASIC, SIMPLE_MUSTACHE_NAME_ID, HbsBuiltinHelperCompletionProvider(
                 "action", "component", "debugger", "get", "if", "input", "link-to", "loc", "log",
-                "outlet", "partial", "render", "textarea", "unbound"))
+                "outlet", "partial", "readonly", "render", "textarea", "unbound"))
```

Comment:
is `{{readonly something}}` a valid use case? I thought it was similar to `mut` and you only use it inside of a subexpression like `{{some-component (readonly something)}}`